python/konduit_cli.md

```diff
@@ -0,0 +1,90 @@
+# Konduit CLI for quick Python experiments
+
+Install with `python setup.py install` from this folder. This will expose a CLI tool called
+`konduit-python`. Assuming you have put a `konduit.jar` in the `tests` folder as described
+in the main `README.md`, you can test this tool as follows:
+
+```bash
+cd tests
+konduit-python --help
+```
+
+which should prompt all currently available commands, i.e.
+
+```bash
+Usage: konduit-python [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  --help  Show this message and exit.
```

Comment:
@maxpumperla can you extend this description to mention the ability to see help for individual sub command?

Reply:
also we want to have some arguments as `required=True` such as the `--os` option for `build` and `init`

Reply:
I do that below